modules/040-node-manager/docs/FAQ.md

```diff
@@ -905,4 +905,109 @@ Once the `worker-spot` NodeGroup reaches its maximum (5 nodes in the example abo
 
+## How to interpret Node Group states?
+
+**Ready** - the node group contains the minimum required number of scheduled nodes with the status ```Ready``` for all zones.
+
+Example 1. A group of nodes in the ``Ready`` state:
+
+```yaml
+apiVersion: deckhouse.io/v1
+kind: NodeGroup
+metadata:
+  name: ng1
+spec:
+  nodeType: CloudEphemeral
+  cloudInstances:
+    maxPerZone: 5
+    minPerZone: 1
+status:
+  conditions:
+  - status: "True"
+    type: Ready
+---
+apiVersion: v1
+kind: Node
+metadata:
+  name: node1
+  labels:
+    node.deckhouse.io/group: ng1
+status:
+  conditions:
+  - status: "True"
+    type: Ready
+```
+
+Example 2. A group of nodes in the ``Not Ready`` state:
+
+```yaml
+apiVersion: deckhouse.io/v1
+kind: NodeGroup
+metadata:
+  name: ng1
+spec:
+  nodeType: CloudEphemeral
+  cloudInstances:
+    maxPerZone: 5
+    minPerZone: 2
+status:
+  conditions:
+  - status: "False"
+    type: Ready
+---
+apiVersion: v1
+kind: Node
+metadata:
+  name: node1
+  labels:
+    node.deckhouse.io/group: ng1
+status:
+  conditions:
+  - status: "True"
+    type: Ready
+```
+
+**Updating** - a node group contains at least one node in which there is an annotation with the prefix ```update.node.deckhouse.io```, for example, ```update.node.deckhouse.io/waiting-for-approval```
+
+**WaitingForDisruptiveApproval** - a node group contains at least one node that has an annotation ```update.node.deckhouse.io/disruption-required``` and
+there is no annotation ```update.node.deckhouse.io/disruption-approved```
+
+**Scaling** - calculated only for node groups with the type ```CloudEphemeral```. The state ```True``` can be in two cases:
+
+1. When the number of nodes is less than the desired number of nodes in the group, i.e. when it is necessary to increase the number of nodes in the group.
+1. When a node is marked for deletion or the number of nodes is greater than the desired number of nodes, i.e. when it is necessary to reduce the number of nodes in the group.
+
+The desired number of nodes is the sum of all replicas in the node group.
+
+Example. The desired number of nodes is 2
+
+```yaml
+apiVersion: deckhouse.io/v1
+kind: NodeGroup
+metadata:
+  name: ng1
+spec:
+  nodeType: CloudEphemeral
+  cloudInstances:
+    maxPerZone: 5
+    minPerZone: 2
+status:
+...
+  desired: 2
+...
+```
+
+**Error** - contains the last error that occurred when creating a node in a node group
+
+## How do I make Werf ignore the Ready conditions in a node group?
+
+Werf checks the ```Ready``` status of resources and, if available, waits for the value to become ```True```. If a large number of nodes are planned in a node group, it may take considerable time to deploy them. As a result, Werf may terminate its work with a timeout error. In order for Werf to ignore the ```Ready``` state of a node group, annotations must be added:
```

Comment:
``` suggestion
[werf](werf.io) checks the ```Ready``` status of resources and, if available, waits for the value to become ```True```. If a large number of nodes are planned in a node group, it may take considerable time to deploy them. As a result, Werf may terminate its work with a timeout error. In order for Werf to ignore the ```Ready``` state of a node group, annotations must be added:
```